### PR TITLE
Port to tidyselect

### LINKED ISSUE #2896: Use tidyselect

### PR DESCRIPTION
**Pull request by lionel-**
Closes #2896

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -9,6 +9,12 @@
 
 * Move build-time vs. run-time checks out of `.onLoad()` and into `dr_dplyr()`.
 
+* dplyr now depends on the new tidyselect package to power `select()`,
```
Comment (hadley) [inline NEWS.md]: This is in the wrong place now

**Comment by lionel-** [comment]
@hadley Tests pass locally. Should we merge?

**Comment by hadley** [comment]
I prefer to wait for a bigger release. We're just doing a patch right now.

**Comment by lionel-** [comment]
I'll disable the dplyr check in tidyselect then. The sentinel value thing didn't work because the dplyr namespace is not populated when the tidyselect onLoad hook is run, so we'll have to check for a hardcoded version.

**Pull request merged**